releasenotes/notes/allowIntegerforQuantumRegisterSize-2e9a6cc033bc962e.yaml

```
@@ -0,0 +1,13 @@
+---
+prelude: >
+    allow use of type integer for for QuantumRegister size 
+
+issues:
+  - |
+    QuantumRegister allows only int, slice, list to denote size
+
+fixes:
+  - |
+    Added numbers.Integral as another allowable type to denote size of QuantumRegister
```

You can use markdown formatting here 👍 
```suggestion
    Use numbers.Integral instead of Python's built-in ``int`` as 
    input type to denote the size of a ``QuantumRegister``.
```